pkgs/development/python-modules/triton/default.nix

```diff
@@ -30,33 +29,20 @@ let
 in
 buildPythonPackage rec {
   pname = "triton";
-  version = "2.1.0";
+  version = "3.0.0";
   pyproject = true;
 
   src = fetchFromGitHub {
-    owner = "openai";
+    owner = "triton-lang";
     repo = pname;
```

Comment:
Keep `repo` disentangled from `pname`

Reply:
thats actually from your original code xD. but I can fix it.

Reply:
Thanks